AGENTS.md

```
@@ -2,657 +2,294 @@
+## Git workflow
 
-The core generates a bearer token at startup written to `{workspace_dir}/core.token` (default `~/.openhuman-staging/core.token` when `OPENHUMAN_APP_ENV=staging`). Read that file for authenticated RPC calls:
+Contribute via your fork. Recommended remotes:
 
-```bash
-TOKEN=$(cat ~/.openhuman-staging/core.token)
-curl http://localhost:7788/rpc -X POST \
-  -H 'Content-Type: application/json' \
-  -H "Authorization: Bearer $TOKEN" \
-  -d '{"jsonrpc":"2.0","method":"core.ping","params":{},"id":1}'
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)